docs/framework/30scoring/operational.md

```diff
@@ -27,4 +29,33 @@ Operational Ability to Pause Score = (O001 + O003 + O004) x O005
 Rationale for scoring:
 
 * The ideal configuration is Role Based Access Control, with active monitoring and a geographically dispersed support team. For decentralized projects, in addition to the automated monitoring system, pausing could be controlled a multisig wallet behind one or more of the accounts of the Role Based Access Control.
-* Shared keys indicates poor security management. This indicates that there may be other poor practices in the system that are not known.
\ No newline at end of file
+* Shared keys indicates poor security management. This indicates that there may be other poor practices in the system that are not known.
+
+
+
+
+### Vulnerability Response Planning Score
+Vulnerability Response Planning is defined by the ability to meet the properties defined in the 
+[Vulnerability Response Planning](../20categories/40operation/vulnerability.md#vulnerability-response-plan) section. The equation for the *Vulnerability Response Planning Score* is:
+
+```
+Vulnerability Response Planning Score = O501 + O502 + O503 + O504
+```
+
+The *Vulnerability Response Planning Score* ranges from 0 to 10.
+
+
+|Question ID  | Question                     |
+|-------------|------------------------------|
+| O501        | Is there a defined *Vulnerability Response Virtual Team*? If yes, score 1, otherwise score 0. |
+| O502        | Is there a vulnerability reporting mechanism that allows issues to be reported without making them public? If yes, score 3, otherwise score 0. |
+| O503        | Is there a defined triage process for vulnerabilities?  If yes, score 1, otherwise score 0. |
+| O504        | Can vulnerabilities fixes be deployed without needing to put code into a public repository? If yes, score 4, otherwise score 0. |
+| O505        | Are *Root Cause Analyses* published? If yes, score 1, otherwise score 0. |
```

Comment:
would protocols that haven't had an incident and thus haven't needed to publish RCAs get a 0 here?